applications/crossbar/test/cb_storage_test.erl

```diff
@@ -3,13 +3,57 @@
 -include_lib("eunit/include/eunit.hrl").
 -include("cb_token_restrictions_test.hrl").
 
+-define(ACCOUNT_ID_UNDER_TEST, <<"account0000000000000000000000001">>).
+-define(ACCOUNT_DB_UNDER_TEST, <<"account%2Fac%2Fco%2Funt0000000000000000000000001">>).
+
+-define(LEGACY_VIEW, <<"vmboxes/legacy_msg_by_timestamp">>).
+
 %% =======================================================================================
-%% Generators
+%% Fixtures / Generators
 %% =======================================================================================
 maybe_check_storage_settings_test_() ->
+    {setup
+    ,fun setup/0
+    ,fun cleanup/1
+    ,fun (_ReturnOfSetup) ->
+             maybe_check_storage_settings()
+     end
+    }.
+
+%% =======================================================================================
+%% Setup
+%% =======================================================================================
+setup() ->
+    ?debugMsg(":: Setting up Kazoo FixtureDB"),
+    %% Required for s3 attachment handler
+    _ = application:ensure_all_started(erlcloud),
+    Pid = kz_fixturedb_util:start_me(),
+    meck:new(kz_datamgr, [unstick, passthrough]),
+    meck:expect(kz_datamgr, get_results, check_kz_datamgr_history()),
+    %% Fix `kz_att_util:format_url/3' issue
+    meck:expect(kz_datamgr, open_cache_doc, fun open_cache_doc/2),
```

Comment:
I'm think instead of mecking `kz_datamgr:open_cache_doc/2` meck `kz_fixturedb_doc:open_doc`.